src/Raven.Studio/typescript/viewmodels/database/query/query.ts

```diff
@@ -184,6 +186,12 @@ class query extends viewModelBase {
     showTimeSeriesGraph: KnockoutComputed<boolean>;
     showPlotButton: KnockoutComputed<boolean>;
     
+    isSpatialQuery = ko.observable<boolean>();
+    spatialDataFound = ko.observable<boolean>();
+    spatialMap = ko.observable<spatialQueryMap>();
+    showMapView: KnockoutComputed<boolean>;
+    viewOnMapClicked = ko.observable<boolean>();
```

Comment:
I don't think we need that field? What if someone clicks on the map twice?

Reply:
Removed the _View on Map_ button.
Removed this field as well.